Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG ENVOY_VERSION
+FROM envoyproxy/envoy-debug:$ENVOY_VERSION AS envoy
+
+FROM ubuntu:18.04 AS perf_data_converter
+
+RUN apt update && apt install -y npm g++ git libelf-dev libcap-dev
+RUN npm install -g @bazel/bazelisk
+
+RUN git clone https://github.com/google/perf_data_converter.git /usr/src/perf_data_converter
+WORKDIR /usr/src/perf_data_converter
+
+RUN bazel build src:perf_to_profile
+RUN cp bazel-bin/src/perf_to_profile /usr/bin/.
```

Comment:
Oh it's just a mannerism I developed over the years when using `cp` and `mv. If I made a mistake and the destination directory didn't exist (e.g. typo or wrong relative path) I want my cp / mv command to fail with `No such file or directory` instead of copying / moving to a file with that bad name.